app/models/manageiq/providers/ansible_tower/automation_manager/configuration_script/api_create.rb

```diff
@@ -2,21 +2,23 @@ class ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScript
   module ApiCreate
     def create_in_provider(manager_id, params)
       manager = ExtManagementSystem.find(manager_id)
+      raise "authentication status is not ok" unless manager.authentication_status_ok?
```

Comment:
Why do we need this?  Shouldn't `with_provider_connection` raise and if that somehow got through, shouldn't `create!` raise?